cards/the_coin/the_coin.gd

```diff
@@ -8,7 +8,7 @@ extends Blueprint
 func setup() -> void:
 	cast_particles.emitting = false
 	
-	card.add_ability(Card.Ability.CAST, cast)
+	card.add_ability(&"Cast", cast)
```

Comment:
Using `StringName` instead of an enum makes it a lot easier to mistype. Figure out if there is some way to prevent that.

Reply:
Making them capitalized makes it worse. Maybe think about making it lowercase / uppercase.